modules/Behavior.js

```diff
@@ -0,0 +1,66 @@
+var _ = require('underscore');
+var TorsoCell = require('./Cell');
+
+// Map of eventName: lifecycleMethod
+var eventMap = {
+  'before-attach': '_attached',
+  'before-detach':  '_detached',
+  'before-activate': '_activate',
+  'before-deactivate': '_deactivate',
+  'before-dispose': '_dispose',
+  'render:before-attach-tracked-views': 'attachTrackedViews',
+  'render:begin': 'prerender',
+  'render:complete': 'postrender',
+  'initialize:begin':  '_preinitialize',
+  'initialize:complete': '_postinitialize'
+};
+
+module.exports = TorsoCell.extend({
+  constructor: function(options) {
+    options = options || {};
+    this.view = options.view;
+    this._bindLifecycleMethods();
+    this._bindEventCallbacks();
+    TorsoCell.apply(this, arguments);
+  },
+
+  _bindLifecycleMethods: function() {
```

Comment:
I think these private methods should be `__` prefixed instead of `_` as they are Torso private methods.